guides-infrastructure-advanced.md

```diff
@@ -0,0 +1,272 @@
+  qm set 9002 --ostype l26
+  qm template 9002
+  ```
+
+### Manually creating Virtual Machines
+Most of our VMs are provisioned with Ansible but sometimes we need to create VMs manually for testing or without automation.
+To create VMs manually follow these steps (the steps below will need to be done over VPN):
+
+#### Select a VM ID, IP address, and DNS name
+1. For the VM you want to create, decide which is the primary category by looking at this table of <a href="https://hackmd.io/pElXZTnUTRO1zApxpdBWDw#VM-ID-categories" data-proofer-ignore>VM ID categories</a>. Then choose a new VM ID for your VM by incrementing the last VM ID in that category. You can look at this table for current <a href="https://github.com/hyphacoop/inventory-private/blob/master/inventory-kvm1.md" data-proofer-ignore>VM ID usage</a>. **Please make sure you add the new VM ID to the table so that it stays up to date.**
+2. Fill out interface(s) <a href="https://hackmd.io/pElXZTnUTRO1zApxpdBWDw#vmbr1-IP-Allocation-internal" data-proofer-ignore>IP allocations</a>
+3. Make a name for internal DNS if needed example `service.hypha.prod` in <a href="https://hackmd.io/pElXZTnUTRO1zApxpdBWDw#Local-DNS" data-proofer-ignore>Internal DNS table</a>
+
+#### Reserve DNS name using pfSense
+4. Log in to pfSense web dashboard <a href="https://198.51.100.1/services_unbound.php" data-proofer-ignore>here</a>. Username `admin`, password is in Passbolt [here](https://pass.hypha.coop/app/passwords/view/c5e705f6-715c-463f-8438-4648ca383fbc)
+5. Add DNS hostname to pfSense's DNS resolver `Host Overrides` under `Services->DNS Resolver` <a href="https://198.51.100.1/services_unbound.php" data-proofer-ignore>here</a>
+- With the example `service.hypha.prod` Host will be `service` Parent domain of host will be `hypha.prod` IP to return for host will be the LAN IPv4 and or IPv6. If dualstack use `,` for comma-separated addresses `10.0.1.9,2001:470:b1f3:1::9`
+
+#### Create a VM using Proxmox
+
+6. To create the VM go login to KVM1 Proxmox host <a href="https://kvm1.hypha.coop:8006/" data-proofer-ignore>here</a> using username root with the password in Passbolt [here](https://pass.hypha.coop/app/passwords/view/a34731d5-eb6a-4f0c-9475-7839280e529b)
+7. Select a template to clone in the `Datacenter` sidebar under `9XXX`
+8. Click on `More` on the top right corner and select `Clone`
+9. `VMID` is the VMID you noted above
+10. Mode must be `Full Clone` 
+11. Choose Target Storage depending where you want the VM to be running from.
+    - local: SSD
+    - local-hdds: Spinning drives
+12. Name should be the public FQDN or *.hypha.prod/stg domain for local access only VMs. 
+13. Format should be qcow2 (default)
+14. Press clone, once finished we can set hardware 
+15. Select the VM ID in the `Datacenter` sidebar and click on `Hardware`
+16. Modify the memory, number of processors and increase diskspace as needed.
+    - Recommended numbers
+        - memory: 1024MB
+        - CPU: 1 socket, 4 cores
```

Comment:
Most servers only need 1 or 2 cores. I would lower this to prevent too high overprovisioning
```suggestion
        - CPU: 1 socket, 2 cores
```